sentry_streams/pyproject.toml

```diff
@@ -7,10 +7,9 @@ name = "sentry-streams"
 version = "0.0.1"
 description = "The python Sentry Streaming API"
 readme = "README.md"
-# Cannot use 3.12 because of pyflink.
-requires-python = ">=3.11,<3.12"
+
+requires-python = ">=3.11"
```

Comment:
Having the flink based adapters and runtime limited to 3.11 but the rest of our application code at 3.13 feels like a runtime difference that will eventually bite us.

Reply:
It can be a deal breaker.  And it is one of the reason I started decoupling it here.
pyflink working properly on 3.13 will be a hard requirement.

I have been looking into the reasons why the wheels are not there above 3.11 and I did not find any evident blocker that we cannot fix. They are mostly due to the library used to run python in thread mode, which we are unlikely to use.
So we will take some time to try to make it run there before ditching it.

Reply:
Perhaps we could help/encourage pyflink to be compatible with python 3.13 🤷